tests/results/TextRangeTest.py

```diff
@@ -7,10 +7,12 @@
 class TextRangeTest(unittest.TestCase):
 
     def test_fail_instantation(self):
-        with self.assertRaises(ValueError):
+        error_message = "End position can't be less than start position"
```

Comment:
can we give this a better name? Also maybe make use of the regex and only do part of the message like `".*less than start.*"` to keep the crucial part or so?